javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+var $countdown = document.querySelector('h1.countdown-display');
+function countDown() {
+  var number = parseInt($countdown.innerHTML);
+  number--;
+  if (number >= 1) {
+    $countdown.textContent = number;
+  } else if (number < 1) {
+    $countdown.textContent = '~Earth Beeeelooowww Us~';
+  }
+}
+
+setInterval(countDown, 2000);
+clearInterval(2);
```

Comment:
Even though the content of the page doesn't seem to change after the countdown reaches `0`, the interval is still continuing to tick away and rewrite the page's content every two seconds (for a demo, `log` out `$countdown.innerHTML` at the beginning of the `countdown` function call). The interval _should_ stop ticking after the countdown reaches `0`:

---

![image](https://user-images.githubusercontent.com/16143771/151103294-999b4b45-6792-4068-bf8f-1d44af29ae0b.png)